cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -1008,6 +1010,10 @@ func getNotRegisteredNodes(allNodes []*apiv1.Node, cloudProviderNodeInstances ma
 	return notRegistered
 }
 
+func withoutErrors(instance cloudprovider.Instance) bool {
```

Comment:
In essence, we want to verify, if we can expect the instance to be Ready or do we see an issue, e.g.
 - instance State is Creating, but ErrorInfo says STOCKOUT

Alternative names: `isUsable`, `canBeRegistered`